static/css/main.css

```diff
@@ -31,78 +31,74 @@ figure {
   margin: 0;
 }
 
-a {
-  color: #279e6a;
-  color: #279e6aee;
+
+.btn {
```

Comment:
I collected your styles to put them into the existing _.button_ class and merge the styles.
https://github.com/neomad-team/neomad.org/blob/master/static/css/main.css#L224